diesel_derives/src/model.rs

```diff
@@ -110,7 +111,18 @@ impl Model {
     }
 
     pub fn table_name(&self) -> &Path {
-        self.table_name.as_ref().unwrap_or(&self.name)
+        match self.table_names.len() {
+            0 => &self.name,
+            1 => &self.table_names[0],
+            _ => abort_call_site!("Multiple table name attributes are not supported"),
```

Comment:
```suggestion
            _ => abort_call_site!("expected a single table name attribute"),
```

Let's avoid erroring out on the call site. Let's try doing it on the spans for table names.

Reply:
ty for the insight. applied